integration-tests/src/test/java/eu/xenit/solr/backup/s3/SolrBackupTest.java

```diff
@@ -137,11 +157,13 @@ void testRestoreEndpoint() {
     @Order(1)
     void testBackupWithNumberToLiveEndpoint() {
         validateSnapshotCount(0);
-        callBackupEndpoint(1);
+        callBackupEndpoint(1, specBackup);
```

Comment:
Maybe call it "request" or "requestSpecification"

or backupspecification, although it is really a request.